addons/ai/CfgWaypoints.hpp

```diff
@@ -0,0 +1,11 @@
+
+class CfgWaypoints {
+    class A3 { // called "Advanced"
+        class CBA_Task_Garrison {
+            displayName = "GARRISON"; // all caps
```

Comment:
was about to write that. unlocalized and all caps. probably to get consistency with the scripting commands. https://community.bistudio.com/wiki/setWaypointType